### DISCUSSION TITLE
preview option is disable

### DISCUSSION THREAD

**Discussion by kostdata** [laravel/nova-issues#5593] [Q&A] [answered]
- Laravel Version: 10.#.#
- Nova Version: v4.24.4 
- PHP Version: 8.2.5
- Database Driver & Version: mysql 8
- Operating System and Version: ubuntu

### Description:

I have Resource Order. 

class Order extends Resource
{
    /**
     * The model the resource corresponds to.
     *
     * @var class-string<\App\Models\Order>
     */
    public static $model = \App\Models\Order::class;
....

in this case there is no action of preview in the table, only editing
![изображение](https://github.com/laravel/nova-issues/assets/91243445/c58785c7-fdeb-465c-9b8a-66b716da902e)
But if I rename model Order  to UserOrder and change resource code, preview option appears in the list

class Order extends Resource
{
    /**
     * The model the resource corresponds to.
     *
     * @var class-string<\App\Models\UserOrder>
     */
    public static $model = \App\Models\UserOrder::class;
.....

![изображение](https://github.com/laravel/nova-issues/assets/91243445/5cf0e6b2-b15a-4fb7-b3d8-d2f4f5a71f6e)

**Comment by crynobone** [comment, marked as answer]
Please check your `App\Models\Order` policy: https://nova.laravel.com/docs/4.0/resources/authorization.html

**Comment by kostdata** [comment]
Thanks
